rules/scala.bzl

```diff
@@ -274,7 +274,17 @@ def make_scala_binary(*extras):
             _extras_attributes(extras),
             *[extra["attrs"] for extra in extras]
         ),
-        doc = "Compiles and links a Scala JVM executable.",
+        doc = """
+Compiles and links a Scala JVM executable.
+
+Produces the following implicit outputs:
+
+  - `<name>_deploy.jar`: a sinlge jar that contains all the necessary information to run the program
```

Comment:
>sinlge

Typo here

I think I'm just commenting on the generated and original sources of this :D